zulipterminal/ui_tools/views.py

```diff
@@ -950,14 +950,29 @@ def keypress(self, size: urwid_Size, key: str) -> str:
 
 class StreamInfoView(PopUpView):
     def __init__(self, controller: Any, color: str,
-                 desc: str, title: str) -> None:
+                 desc: str, stream_id: int, title: str) -> None:
         # Add 4 (for 2 Unicode characters on either side) to the popup title
         # length to make sure that the title gets displayed even when the
         # content is shorter than the title length (+4 Unicode characters).
-        width = max(len(desc) + 2, len(title) + 4)
-        stream_info_content = [urwid.Text(desc, align='center')]
-        super().__init__(controller, stream_info_content, 'STREAM_DESC', width,
-                         title)
+        # 20 is the length required to show stream settings labels.
+        width = max(len(desc) + 2, len(title) + 4, 20)
```

Comment:
I'm not sure how you plan to develop this further, but it may be useful to simply extract the label text used later and place a `len` expression in here, so the code will express what is happening rather than needing another comment.

Reply:
Hmm, I still need a comment to explain why I'm adding 4

Reply:
@kaustubh-nair Which 4?

Reply:
Ah, sorry this is outdated
This is what I'm referring to
```python3
        # Add 4 to stream settings label for accomodating checkbox.
        width = max(len(desc) + 2, len(title) + 4,
                    len(muted_setting.label) + 4)
```

Reply:
The +4 is `len([UNICODE]space)`.

Also, there is a typo in the comment.